### ISSUE TITLE
cursor: initial ?

### ISSUE THREAD

**Issue by matt-bailey** [csstools/sanitize.css#26]
Hi. I've not seen this property on cursor before, is this a typo? On a site I'm working on it's causing some links to have a text selection cursor rather than a pointer (in FireFox - Chrome, however, seems fine).

**Comment by brunorios1** [comment]
Hi @matt-bailey,
The cursor property value is defined to "default" by default:
https://github.com/jonathantneal/sanitize.css/blob/master/sanitize.scss#L4

You overwrote with "initial" value?
Using "auto" is a better choice because "initial" is not supported by Internet Explorer:
https://developer.mozilla.org/en-US/docs/Web/CSS/initial

**Comment by matt-bailey** [comment]
In the version of sanitize we pulled in a few weeks ago using bower this was in the source code (sanitize.scss, lines 137-149):

``` scss
// style cursor to match native experience

* {
    cursor: inherit;
}

a,
button,
input,
select,
textarea {
    cursor: initial;
}
```

However, I can see that this issue with Firefox has already been spotted and fixed 16 days ago [here](https://github.com/jonathantneal/sanitize.css/commit/5bb2143075e633d70f8d071622ac64a53bedaab1#diff-2497a17f2113d43d510afe5128e228dbL139).

I will update the version of sanitize we are using in bower.

I'm still curious as to what `cursor: initial` is/does - I can't see it mentioned anywhere in the spec for `cursor`: https://developer.mozilla.org/en-US/docs/Web/CSS/cursor

**Comment by ungoldman** [comment]
@matt-bailey as @brunorios1 mentioned above it is a real thing (was news to me as well): https://developer.mozilla.org/en-US/docs/Web/CSS/initial

However it looks like @jonathantneal changed it to `cursor: inherit` in the working version on master, so this change will be reflected in the next release.

Leaving this issue open as a reminder until the release happens.

**Comment by jonathantneal** [comment]
I believe this was resolved in the 1.2.0 release. Sorry to only respond now.
